packages/react/src/Bento/Bento.module.css

```
@@ -0,0 +1,1793 @@
+
+  .Bento__Item--xsmall-row-span-9 {
+    grid-row: auto/span 9;
+  }
+
+  .Bento__Item--xsmall-row-span-10 {
+    grid-row: auto/span 10;
+  }
+
+  .Bento__Item--xsmall-row-span-11 {
+    grid-row: auto/span 11;
+  }
+
+  .Bento__Item--xsmall-row-span-12 {
+    grid-row: auto/span 12;
+  }
+
+  /* xsmall Column Start */
+  .Bento__Item--xsmall-column-start-1 {
+    grid-column-start: 1;
+  }
+
+  .Bento__Item--xsmall-column-start-2 {
+    grid-column-start: 2;
+  }
+
+  .Bento__Item--xsmall-column-start-3 {
+    grid-column-start: 3;
+  }
+
+  .Bento__Item--xsmall-column-start-4 {
+    grid-column-start: 4;
+  }
+
+  .Bento__Item--xsmall-column-start-5 {
+    grid-column-start: 5;
+  }
+
+  .Bento__Item--xsmall-column-start-6 {
+    grid-column-start: 6;
+  }
+
+  .Bento__Item--xsmall-column-start-7 {
+    grid-column-start: 7;
+  }
+
+  .Bento__Item--xsmall-column-start-8 {
+    grid-column-start: 8;
+  }
+
+  .Bento__Item--xsmall-column-start-9 {
+    grid-column-start: 9;
+  }
+
+  .Bento__Item--xsmall-column-start-10 {
+    grid-column-start: 10;
+  }
+
+  .Bento__Item--xsmall-column-start-11 {
+    grid-column-start: 11;
+  }
+
+  .Bento__Item--xsmall-column-start-12 {
+    grid-column-start: 12;
+  }
+
+  /* xsmall Row Start */
+  .Bento__Item--xsmall-row-start-1 {
+    grid-row-start: 1;
+  }
+
+  .Bento__Item--xsmall-row-start-2 {
+    grid-row-start: 2;
+  }
+
+  .Bento__Item--xsmall-row-start-3 {
+    grid-row-start: 3;
+  }
+
+  .Bento__Item--xsmall-row-start-4 {
+    grid-row-start: 4;
+  }
+
+  .Bento__Item--xsmall-row-start-5 {
+    grid-row-start: 5;
+  }
+
+  .Bento__Item--xsmall-row-start-6 {
+    grid-row-start: 6;
+  }
+
+  .Bento__Item--xsmall-row-start-7 {
+    grid-row-start: 7;
+  }
+
+  .Bento__Item--xsmall-row-start-8 {
+    grid-row-start: 8;
+  }
+
+  .Bento__Item--xsmall-row-start-9 {
+    grid-row-start: 9;
+  }
+
+  .Bento__Item--xsmall-row-start-10 {
+    grid-row-start: 10;
+  }
+
+  .Bento__Item--xsmall-row-start-11 {
+    grid-row-start: 11;
+  }
+
+  .Bento__Item--xsmall-row-start-12 {
+    grid-row-start: 12;
+  }
+
+  /* XSMALL Flow */
+  .Bento__Item--xsmall-flow-column {
+    grid-auto-flow: column;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  .Bento__Item--xsmall-flow-row {
+    grid-auto-flow: row;
+    grid-template-rows: 1fr 1fr;
+    grid-template-columns: 1fr;
+  }
+
+  /* XSMALL Vertical Align */
+  .Bento__Item--xsmall-verticalAlign-start {
+    justify-content: start;
+  }
+
+  .Bento__Item--xsmall-verticalAlign-center {
+    justify-content: center;
+  }
+
+  .Bento__Item--xsmall-verticalAlign-end {
+    justify-content: end;
+  }
+
+  /* XSMALL Horizontal Align */
+  .Bento__Item--xsmall-horizontalAlign-start {
+    align-items: start;
+  }
+
+  .Bento__Item--xsmall-horizontalAlign-center {
+    align-items: center;
+  }
+
+  .Bento__Item--xsmall-horizontalAlign-end {
+    align-items: end;
+  }
+}
+
+/* SMALL breakpoint */
+@media screen and (min-width: 0) {
```

This value should be `34rem` for the small breakpoint, currently the same as xsmall.

```suggestion
@media screen and (min-width: 34rem) {
```

---

This has been updated in the following SHA: https://github.com/primer/brand/pull/390/commits/015e64c3e342f101c896f8259daf2c49feb49558